packages/legend-graph/src/models/metamodels/pure/packageableElements/service/ServiceTest_Legacy.ts

```diff
@@ -0,0 +1,124 @@
+/**
+ * Copyright (c) 2020-present, Goldman Sachs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { type Hashable, hashArray, uuid } from '@finos/legend-shared';
+import { CORE_HASH_STRUCTURE } from '../../../../../MetaModelConst';
+import type { RawLambda } from '../../rawValueSpecification/RawLambda';
+import type { Service } from './Service';
+
+/**
+ * TODO: Remove once migration from `ServiceTest_Legacy` to `ServiceTest` is complete
+ * @deprecated
+ */
+export abstract class ServiceTest_Legacy implements Hashable {
```

Comment:
Can we make the effort to change this to `DEPREACTED__ServiceTest`. Thanks!

Might need to apply this to my other comments.

Reply:
Also remember to change the name of the files